src/api/platform/metadata/push_request.rs

```diff
@@ -70,12 +72,18 @@ impl PushRequest {
 
             stream_body: Some(stream_body),
 
-            valid_keys,
+            verifying_keys,
             deleted_block_cids,
         })
     }
 }
 
+#[derive(Debug, Serialize)]
+pub struct PushKey {
+    fingerprint: String,
+    pem: String,
```

Comment:
pem is format not a descriptive name, please continue to use "public_key". We wouldn't want a variable just named "json", this is the equivalent.